src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/BarnFyllerÅrOppfølgingsoppgaveService.kt

```diff
@@ -84,7 +83,7 @@ class BarnFyllerÅrOppfølgingsoppgaveService(
 
     private fun opprettOppgaveForBarn(barn: List<BarnTilUtplukkForOppgave>): List<OpprettOppgaveForBarn> =
         barn.mapNotNull {
-            Alder.fromFødselsdato(fødselsdato(it))?.let { alder ->
+            Alder.fromFødselsdato(it.termindatoBarn)?.let { alder ->
```

Comment:
Dette funker bare for de som er terminbarn, men flertallet er ikke dét. 
Vi burde nok heller vurdere å hente ut "fødsel" fra PDL for alle disse barna. Burde kunne gjøres med et kall mot PDL hvis vi vet fødselsnummer til barna?